qa/rpc-tests/rpcbind_test.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python3
 # Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2020 The Zcash developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://www.opensource.org/licenses/mit-license.php .
 
 # Test for -rpcbind, as well as -rpcallowip and -rpcconnect
 
-# Dependency: python-bitcoinrpc
+# Dependency: python-zcashrpc
```

Comment:
Is this a correct change?

Reply:
I suspect you're right, I will examine the git history.

Reply:
Reverted.